pipeline_products_session/jwst-data-products-part2.ipynb

```diff
@@ -0,0 +1,1401 @@
+    "\n",
+    "## 6.1.<font color='white'>-</font>Imaging<a class=\"anchor\" id=\"s2-imaging\"></a>\n",
+    "\n",
+    "Stage 2 image processing applies additional instrumental corrections and calibrations that result in a fully calibrated individual exposure. Non-time series exposures use the CALWEBB_IMAGE2 configuration, which applies all applicable steps to the data. The CALWEBB_TSO-IMAGE2 configuration, on the other hand, should be used for time series exposures, for which some steps are set to be skipped by default. Both configurations call the Image2Pipeline; the only difference is which steps are applied.\n",
+    "\n",
+    "### A.<font color='white'>-</font>Input<a class=\"anchor\" id=\"s2-imaging-input\"></a>\n",
+    "\n",
+    "The inputs to this stage are listed below.\n",
+    "\n",
+    "* **2D or 3D countrate data**\n",
+    "    * **Data model**: ImageModel or CubeModel\n",
+    "    * **File suffix**: ```_rate``` or ```_rateints```\n",
+    "    * **Description**: The input to Image2Pipeline is a countrate exposure, either ```_rate``` or ```_rateints``` data. A single input file can be processed or an association file listing multiple inputs can be used, in which case the processing steps will be applied to each input exposure, one at a time. If ```_rateints``` products are used, each step applies its algorithm to each integration in the exposure. Time series and coronagraphic exposures are expected to use 3D data as input, to be processed on a per-integration basis. Time series exposures will use the CALWEBB_TSO2 configuration, while coronagraphic exposures should use the CALWEBB_IMAGE2 configuration.\n",
+    "    \n",
+    "\n",
+    "\n",
+    "### B.<font color='white'>-</font>Output<a class=\"anchor\" id=\"s2-imaging-output\"></a>\n",
+    "\n",
+    "The outputs of this stage are listed below.\n",
+    "\n",
+    "* **2D or 3D calibrated data**\n",
+    "    * **Data model**: ImageModel or CubeModel\n",
+    "    * **File suffix**: ```_cal``` or ```_calints```\n",
+    "    * **Description**: The output is a fully calibrated, but unrectified, exposure, using the product type suffix ```_cal``` or ```_calints```, depending on the type of input.\n",
+    "    \n",
+    "    \n",
+    "* **2D resampled image**\n",
+    "    * Data model: DrizProductModel\n",
+    "    * File suffix: ```_i2d```\n",
+    "    * Description: This is the output of the resample step and is only created for regular direct imaging observations (not for time series or coronagraphy 3D data sets). Note that this product is intended for quick-look use only and is not passed along as input to Stage 3 processing. Calibrated, but unrectified (```_cal```) products are used as input to Stage 3. \n",
```

Comment:
It might be good to quickly define "resampled" here

Reply:
Fixed